src/crypto/CHIPCryptoPALPSA.cpp

```diff
@@ -698,22 +712,101 @@ CHIP_ERROR P256Keypair::Initialize(ECPKeyTarget key_target)
     return error;
 }
 
+CHIP_ERROR P256Keypair::LoadKeypairFromRaw(ByteSpan private_key, ByteSpan public_key)
+{
+    CHIP_ERROR error                = CHIP_NO_ERROR;
+    psa_key_attributes_t attributes = configure_ecc_key_pair_attributes();
+    psa_status_t status;
+    PsaP256KeypairContext & context = ToPsaContext(mKeypair);
+
+    status = psa_import_key(&attributes, private_key.data(), kP256_PrivateKey_Length, &context.key_id);
+    VerifyOrExit(status == PSA_SUCCESS, error = CHIP_ERROR_INTERNAL);
+
+    memcpy(mPublicKey.Bytes(), public_key.data(), kP256_PublicKey_Length);
+    memcpy(mKeypair.mBytes, &context.key_id, sizeof(psa_key_id_t));
+    mInitialized = true;
+
+exit:
+    logPsaError(status);
+
+    return error;
+}
+
+CHIP_ERROR P256Keypair::LoadKeypairFromRaw(const uint8_t * private_key, const size_t private_key_size, const uint8_t * public_key,
+                                           const size_t public_key_size)
+{
+    psa_key_attributes_t attributes = configure_ecc_key_pair_attributes();
+    psa_key_id_t key_id;
+    psa_status_t status;
```

Comment:
Don't we need to use the PSA context here as well?

Reply:
No because that is already under the PSA source :)